lib/ueberauth/plug.ex

```diff
@@ -203,13 +201,13 @@ defmodule Ueberauth.Plug do
   defp validate_config!([]), do: raise_invalid_config!()
   defp validate_config!(config) do
     case Keyword.get(config, :providers) do
-      providers when is_list(providers) ->
-        if length(providers) == 0, do: raise_invalid_config!()
+      [] -> raise_invalid_config!()
```

Comment:
`list == []`, `Enum.empty?/1`, or pattern matching on `[]` is preferred to `length(list) == 0`